plugins/include/tse.inc

```diff
@@ -0,0 +1,266 @@
+	TSE_WPN_M4A1,
+	TSE_WPN_MP5SD,
+	TSE_WPN_MP5K,
+	TSE_WPN_ABERETTAS,
+	TSE_WPN_MK23,
+	TSE_WPN_AMK23,
+	TSE_WPN_USAS,
+	TSE_WPN_DEAGLE,
+	TSE_WPN_AK47,
+	TSE_WPN_57,
+	TSE_WPN_AUG,
+	TSE_WPN_AUZI,
+	TSE_WPN_SKORP,
+	TSE_WPN_M82A1,
+	TSE_WPN_MP7,
+	TSE_WPN_SPAS,
+	TSE_WPN_GCOLTS,
+	TSE_WPN_GLOCK20,
+	TSE_WPN_UMP,
+	TSE_WPN_M61GRENADE,
+	TSE_WPN_CKNIFE,
+	TSE_WPN_MOSSBERG,
+	TSE_WPN_M16A4,
+	TSE_WPN_MK1,
+	TSE_WPN_C4,
+	TSE_WPN_A57,
+	TSE_WPN_RBULL,
+	TSE_WPN_M60E3,
+	TSE_WPN_SAWED_OFF,
+	TSE_WPN_KATANA,
+	TSE_WPN_SKNIFE,
+	TSE_WPN_CONTENDER,
+	TSE_WPN_ASKORP
+};
+
+// Forwards
+
+/* Called when a player was stunted */
+forward client_onstunt(idx, stunttype);
+
+/* Called when a player makes melee hit */
+forward client_onmeleehit(idx, Float:damage, Float:duration);
+
+/* Called when a player picks up powerup */
+forward client_onpickuppwup(idx, type, duration);
+
+// Returns value of free player slots (weight), on error returns -1
+native tse_getuserslots(idx);
+
+// Sets value of free player slots (weight), on error returns 0
+native tse_setuserslots(idx, val);
+
+// Returns value of player cash, on error returns -1
+native tse_getusercash(idx);
+
+// Sets value of player cash, on error returns 0
+native tse_setusercash(idx, val);
+
+// Gets if player has a kevlar (returns 1), or not (returns 0), on error returns -1
+native tse_getuserkevlar(idx);
+
+// Returns user stamina count, on error returns -1
+native tse_getuserstamina(idx);
+
+// Sets user stamina count, on error returns 0
+native tse_setuserstamina(idx, Float:val);
+
+// Returns float value of effects physics speed (speed of sparks, bullets, surrounding players sounds, broken glass pieces and other particles), on error returns -1
+native tse_geteffectsphysspd(idx);
+
+// Sets float value of effects physics speed, on error returns 0
+native tse_seteffectsphysspd(idx, Float:val);
+
+// Returns float value of player physics speed (speed of animation, movement, surrounding particles sounds, stunting, throwed knives), on error returns -1
+native tse_getuserphysspd(idx);
+
+// Sets float value of player physics speed, on error returns 0
+native tse_setuserphysspd(idx, Float:val);
+
+/* 
+*  Sets weapon shooting params for specified weapon of specified player, on error returns 0 
+*  weapid - weapon ID (TSE_WPN_*)
+*  Float:acc - float value of accuracy
+*  Float:vkick - float value of vertical kick
+*  Float:acckick - float value of accuracy kick
+*  Float:dmg - float value of damage
+*  cost - cost of weapon in shop
+*/
+native tse_sendweapparams(idx, weapid, Float:acc, Float:vkick, Float:acckick, Float:dmg, cost);
+
+// Returns player current items (TSE_ITEM_*), on error returns -1
+// WARNING: If you want check presence of who items, use them sum
+native tse_getusercurritems(idx);
+
+// Sets player current items (TSE_ITEM_*), on error returns 0
+native tse_setusercurritems(idx, val);
+
+// Returns player current weapon ID (TSE_WPN_*), on error returns -1
+native tse_getusercurrweap(idx);
```

Comment:
Create tag for enum like
```
enum MY_ENUM
{
  ITEM1,
  ITEM2,
}
```
Then use it in functions.
```
native MY_ENUM:tse_makemegod(idx);
```

Reply:
This method is likely to be true, but it is unusual and is rarely used in Pawn (for example see engine_const.inc, fakemeta_const.inc). Anyway, thanks for your helping!